addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset ...attributes class='rose-form-fieldset'>
+    <legend 
+        id={{concat 'title-' this.id}}
+        aria-describedby={{concat 'description-' this.id}}
```

Comment:
`describedby` would go on the `fieldset`.

Reply:
@randallmorey Do you mean the `aria-describedby` pertains to the fieldset and then the `aria-labelledby` should be move to legend?

Reply:
Fieldset should have the `aria-describedby` attribute.  And we can probably eliminate the `aria-labelledby`?  `legend` semantically serves the purpose so there should be no ambiguity to assistive tech.

Reply:
Ahh gotcha that makes sense